extensions/wikia/EmbeddableDiscussions/models/DiscussionsThreadModel.class.php

```diff
@@ -43,7 +44,18 @@ private function getUpvoteRequestUrl( $id ) {
 	}
 
 	private function apiRequest( $url ) {
-		$data = Http::get( $url );
+		global $wgUser;
+
+		$request = $wgUser->getRequest();
```

Comment:
If you're just getting the request object, `RequestContext::getMain()->getRequest();` would be better than getting it from the global user object.